snipeit.go

```diff
@@ -216,8 +219,19 @@ func (c *Client) Location(id int64) (*Location, *http.Response, error) {
 // HardwareOptions specifies a subset of optional query parameters
 // for listing assets.
 type HardwareOptions struct {
-	LocationID int `url:"location_id,omitempty"`
-	CompanyID  int `url:"company_id,omitempty"`
+	Limit          int    `url:"limit,omitempty"`
+	Offset         int    `url:"offset,omitempty"`
+	Search         string `url:"search,omitempty"`
+	Ordernumber    string `url:"order_number,omitempty"`
```

Comment:
Use camelcase for variable names composed from two or more words. e.g. `OrderNumber`